docs/.vitepress/vitepress/components/navbar/vp-search.vue

```diff
@@ -52,24 +52,24 @@ function update(options: any) {
   }
 }
 
-const lang = useLang()
+// const lang = useLang()
 
 function initialize(userOptions: any) {
   // if the user has multiple locales, the search results should be filtered
   // based on the language
-  const facetFilters = props.multilang ? [`language:${lang.value}`] : []
+  // const facetFilters = props.multilang ? [`language:${lang.value}`] : []
 
   docsearch(
     Object.assign({}, userOptions, {
       container: '#docsearch',
       indexName: 'element-plus',
-      searchParameters: Object.assign({}, userOptions.searchParameters, {
-        // pass a custom lang facetFilter to allow multiple language search
-        // https://github.com/algolia/docsearch-configs/pull/3942
-        facetFilters: facetFilters.concat(
-          userOptions.searchParameters?.facetFilters || []
-        ),
-      }),
+      // searchParameters: Object.assign({}, userOptions.searchParameters, {
+      //   // pass a custom lang facetFilter to allow multiple language search
+      //   // https://github.com/algolia/docsearch-configs/pull/3942
+      //   facetFilters: facetFilters.concat(
+      //     userOptions.searchParameters?.facetFilters || []
+      //   ),
+      // }),
```

Comment:
Just remove them

Reply:
This is just a temp fix.

I'll try to update the searchParameters logic to make it really work.